### PR TITLE
fix: Issue in list_array.cpp resolved

### PR DESCRIPTION
**Pull request by sagarpandyansit**
#### Issue that resolved
https://github.com/TheAlgorithms/C-Plus-Plus/issues/1338
#### Description of Change
- Before there is no code written in the list of operations to exit from the infinite while loop.
- I added the exit code in the list.
- I added appropriate comment for the list.

#### Checklist

- [ ] Added description of change
- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [ ] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1342"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please fix [`clang-tidy`](https://github.com/TheAlgorithms/C-Plus-Plus/pull/1342/checks?check_run_id=1276731205) warnings.

File: data_structures/list_array.cpp [hunk 1]
```diff
@@ -129,28 +132,32 @@ int main() {
...
+            case 5:
+                L.Show();
+                break;
+            default:
+                std::cout << "\nplease enter valid option.";
         }
     } while (choice != 0);
     return 0;
-}
+}
```
Comment (Panquesito7) [inline data_structures/list_array.cpp:163]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — COMMENTED** [review]
We can take the opportunity to add proper documentation and refined code. 🙂
Confused? See the [typical structure](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) of a program or see #962.

**Comment by sagarpandyansit** [comment]
@Panquesito7 Actually, this branch got some issues, that's why I had made another branch and started work on the refinement of code already.
Can you please merge this pull request? I will make another pull request with the documentation and refined code shortly. Thanks.

**Review by Panquesito7 — APPROVED** [review]
LGTM; good work. 😄👍

**Pull request merged**